src/lib/components/artwork_grids/generic_grid.tsx

```diff
@@ -139,8 +134,8 @@ class GenericArtworksGrid extends React.Component<Props, State> {
 }
 
 interface Styles {
-  container: ViewStyle,
-  section: ViewStyle,
+  container: ViewStyle
```

Comment:
This is actually what we prefer nowadays, left a comment https://github.com/prettier/prettier/issues/1728#issuecomment-304608917.